chirp/locale/es.po

```diff
@@ -6,13 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: CHIRP\n"
 "Report-Msgid-Bugs-To: \n"
-<<<<<<< HEAD
-"POT-Creation-Date: 2025-11-25 21:16-0800\n"
+"POT-Creation-Date: 2025-12-08 20:06-0500\n"
 "PO-Revision-Date: 2025-12-03 22:15-0300\n"
-=======
-"POT-Creation-Date: 2025-12-07 08:11-0800\n"
-"PO-Revision-Date: 2025-09-05 19:57-0400\n"
->>>>>>> 29f4ff47 (Update pt_BR translation from Luiz Ricardo Prado)
```

Comment:
Thanks, this just crept in and would have broken tomorrow's build. So I'm fixing this on master now. You might have to re-fix yours when you rebase.